src/context/WalletProvider/NativeWallet/NativeTestPhrase.tsx

```diff
@@ -62,15 +63,18 @@ export const NativeTestPhrase = ({ history, location }: NativeSetupProps) => {
 
   return !shuffledWords.length ? null : (
     <>
-      <ModalHeader>Lets practice your secret phrase</ModalHeader>
+      <ModalHeader>
+        <Text1 translation={'walletProvider.nTest.header'} />
+      </ModalHeader>
       <ModalBody>
         <Text>
```

Comment:
I know you did not add the Chakra Text component in here. 

But Since we have our Text and RawText components that wrap the Chakra Text component. I think we want to use our wrapped versions of the components so if we need to make any more changes to the wrapped Text components that it will flow through the site easily.

So would you mind updating this to be `RawText` here and then we can change `Text1` to just `Text`